src/components/about/About.js

```diff
@@ -23,8 +17,12 @@ const About = () => (
             Welcome to
           </Section.Text>
           <Section.Row style={styles.imageContainer}>
-            <Image source={splashImage} style={styles.logo} resizeMode="contain" />
-            <Image source={goodDollarImage} style={styles.goodDollar} resizeMode="contain" />
+            <View style={styles.logo}>
```

Comment:
we need to incapsulate styles prop to the component generated from SVG.
it could be done by using custom templates (see links i've send you in Slack).
i'll attach them also here:
https://react-svgr.com/docs/custom-templates/
https://react-svgr.com/docs/custom-transformations/